src/main/resources/idprovider/idprovider.xml

```diff
@@ -54,6 +54,13 @@
       <occurrences minimum="0" maximum="1"/>
     </input>
 
+    <input name="pageSize" type="Long">
+      <label>The page return size from graph api</label>
+      <default>5</default>
+      <help-text>If the result contains more than the page size, graph api will return an @odata.nextLink property similar to the following along with the first page of users.</help-text>
+      <occurrences minimum="0" maximum="1"/>
```

Comment:
Are we sure that we want `minimum="0"`? Might as well make it mandatory. Also the code in *group.js* does not take into account that the value can be missing.

Reply:
Yes, now it does, since page size should be optionally.